examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/src/authorizer/lambda_function.py

```diff
@@ -0,0 +1,218 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# IMPORTANT: 
+#     This is sample implementation of a Lambda Authorizer
+#     It generates IAM policy that allows ALL actions on your API to be performed by ANYONE
+#     Make sure to update code below to limit access to the resources based on your use case
```

Comment:
I've removed all Lambda Authorizer blueprint code from this example and left hardcoded policy only, along with the warning that users should update it based on their use case. Also, I've provided links to the documentation, Lambda Authorizer blueprints in GitHub, and sample implementation for JWT based authorization.

I'd like to keep this example lean and simple, so didn't want to introduce additional complexity of generating random password/token and parameterizing deployments. Any enterprise environment would have their own IdP and requirements for authentication/authorization, most likely JWT based. I'd rather make sure to point users to overall microservice implementation using Crossplane instead and providing references to the full scale implementation in case they need it.   

Let me know if you think fully functioning Lambda Authorizer is necessary, will implement it as you described it in the comment.

Reply:
Thanks for removing the unnecessary code. I agree that JWT is usually how authentication and authorization is done, but I would still argue that we should implement a bare minimum authentication method when creating a public facing endpoint. Especially if this endpoint directly allows you to interact with a database. For example, a malicious actor could easily store illicit materials into the DynamoDB table here as long as the API Gateway URL is known, because we do not sanitize or validate payload. 

Adding randomly generated password client side should be straight forward to implement. See: https://github.com/awslabs/crossplane-on-eks/blob/main/examples/aws-provider/composite-resources/daskhub/README.md

Reply:
I've added randomly generated password to the claim, value is passed to the Lambda Authorizer via environmental variable. I also updated README.md and code comments to indicate that users should implement authorization based on their use case and provided links to the documentation and sample implementations.
Let me know if you thin we need something more elaborate.

Reply:
Updated to use AWS Secrets Manager secure string.